packages/yoroi-extension/chrome/constants.js

```diff
@@ -64,6 +64,6 @@ export function genCSP(request: {|
     `object-src 'self' ${objectSrc.join(' ')};`,
     `connect-src ${connectSrc.join(' ')};`,
     `style-src * ${evalStyle} 'self' ${styleSrc.join(' ')} blob:;`,
-    `img-src 'self' ${imgSrc.join(' ')} data:;`,
+    `img-src 'self' ${imgSrc.join(' ')} data: https://icons.duckduckgo.com/ip3/ ;`,
```

Comment:
Although this is definitely an option, I wonder if it's possible to instead have the connector to get the icon for the page it is one and then send that over to Yoroi instead. Maybe this way we wouldn't need to depend on an external service or modify our CSP

Reply:
@SebastienGllmt I tried to convert from the image url to a data:image format to have like `data:image/png;base64...` from connector, but I got some CORS issues at requesting the image from `icons.duckduckgo` site. Then I tried to give permissions to that url from connector manifest json but still didn't work. Do you have any different idea how to turn off CORS?  
```Access to image at 'https://icons.duckduckgo.com/ip3/localhost.ico' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.```